doc/whats-new.rst

```diff
@@ -189,6 +189,8 @@ Enhancements
 - Allow ``expand_dims`` method to support inserting/broadcasting dimensions
   with size > 1. (:issue:`2710`)
   By `Martin Pletcher <https://github.com/pletchm>`_.
+- New methods for reshaping Datasets of variables with different dimensions
```

Comment:
this needs to move up to 0.12.3 now -- sorry for the churn here!

Reply:
Sure. I moved this up to that release and added a new section header
```
v0.12.3 (unreleased)
--------------------

New functions/methods
~~~~~~~~~~~~~~~~~~~~~

- New methods for reshaping Datasets of variables with different dimensions
  (:issue:`1317`). By `Noah Brenowitz <https://github.com/nbren12>`_.

```